### PR TITLE
[how-to-contribute Page] Translate to Japanese Page

### LINKED ISSUE #38: [how-to-contribute Page] Translate to Japanese
[Contribution]
Translate English to Japanese the how-to-contribute page.

https://vue-a11y.com/jp/project/how-to-contribute.html

### PR DESCRIPTION
**Pull request by kazuki-komori**
close #38 

## sections

- [X] How to contribute
- [X] Development
- [x] Writing
- [x] Answering questions
- [x] Sharing
- [x] Supporting

### CODE REVIEW

**Comment by ktquez** [comment]
Deploy preview
https://deploy-preview-39--vue-a11y.netlify.app/jp/project/how-to-contribute.html

File: src/jp/project/how-to-contribute.md [hunk 1]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+# 貢献する方法
 
-First of all, happy that you are here on this page and thinking about contributing to the project.
+まず、あなたがこのページに訪れ、このプロジェクトに対する貢献について考えてくださることに感謝します。
 
-## Development
+## 開発
 
-To contribute to the development of the site, you can initially open an issue on GitHub and report an error or propose a new feature.
+このサイトの開発に貢献するためには、まず最初に Github 上で issue を開き、エラー報告や新機能の提案をすることができます。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:12]:
```suggestion
このサイトの開発に貢献するためには、まず最初に GitHub 上で issue を開き、エラー報告や新機能の提案をすることができます。
```

File: src/jp/project/how-to-contribute.md [hunk 2]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
 ::: alert tip
-Try to follow the [official style guide for Vue.js](https://vuejs.org/v2/style-guide/) as much as possible.
+[Vue.js の公式スタイルガイド](https://jp.vuejs.org/v2/style-guide/) にできるだけ従って下さい。
 :::
 
-## Writing
+## 執筆
 
-We will love to hear your content idea, so the first step is to open an issue using the specific template.
+あなたのコンテンツに関するアイデアを是非知りたいので、そのための第一段階として特定のテンプレートを用いて issue を開いてください。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:24]:
```suggestion
あなたのコンテンツに関するアイデアを是非知りたいので、その第一歩として特定のテンプレートを用いて issue を開いてください。
```

「第一段階」は硬いのでこちらの言い回しはいかがでしょう

File: src/jp/project/how-to-contribute.md [hunk 3]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
 
-We also have an [issue](https://github.com/vue-a11y/vue-a11y.com/issues/14) with several content ideas, feel free to add other subjects that you want to see as content on the site.
+また、いくつかのコンテンツのアイデアを [issue](https://github.com/vue-a11y/vue-a11y.com/issues/14) にまとめているので、サイト上でコンテンツとして見たいテーマを気軽に追加してください。
 
-Our intention is to reward the creator for each content published on the site, we will soon achieve this feat.
+私たちの意図はサイト上に公開されている各コンテンツの製作者に報いるためであり、間もなくこの偉業を成し遂げられます。
 
 ::: alert warning
-We created a guide with tips for creating posts, recipes, or tips. [Posts guide](/project/posts-guide.html#posts-guide)
+記事やレシピ、tips の作成に関するヒントをまとめたガイドを作成しました。 [投稿ガイド](/project/posts-guide.html#posts-guide)
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:37]:
```suggestion
記事やレシピ、ヒントの作成に関するヒントをまとめたガイドを作成しました。 [投稿ガイド](/project/posts-guide.html#posts-guide)
```

tips => ヒントに翻訳でお願いします 🙏 
https://vue-a11y.com/jp/recipes/tips/aria-attributes-in-vue.html

Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:37]:
```suggestion
記事やレシピ、テクニックの作成に関するヒントをまとめたガイドを作成しました。 [投稿ガイド](/project/posts-guide.html#posts-guide)
```

How about this?

File: src/jp/project/how-to-contribute.md [hunk 4]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+## 質問への回答
 
-In our [GitHub Problem Tracker](https://github.com/vue-a11y/vue-a11y.com/issues), we are open to answering questions from people of all levels of knowledge.
+[GitHub Problem Tracker](https://github.com/vue-a11y/vue-a11y.com/issues) では、私たちはあらゆるレベルの知識を持つ人々からの質問に回答しています。
 
-We will be grateful to receive your help by answering questions and also sharing your experience with us.
+質問に回答したり、経験を共有することで、あなたの手助けを頂ければ幸いです。
 
-We are also daily answering questions in the Vue.js community on Discord ([Vue Land](https://discord.gg/vue)), join us.
+また、 Discord での Vue.js コミュニティで日々質問に回答しています ([Vue Land](https://discord.gg/vue)), 是非、ご参加ください。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:46]:
```suggestion
また、 Discord での Vue.js コミュニティで日々質問に回答しています ([Vue Land](https://discord.gg/vue))、 是非ご参加ください。
```

File: src/jp/project/how-to-contribute.md [hunk 5]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+## 支援
 
-The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+このプロジェクトは、Vue.jsやアクセシビリティを愛し、 Web がより包括的になることの重要性を理解している人々によって支えられています。
 
-Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+皆様からのご支援は非常に重要であり、コンテンツ作成にさらに時間を割くことができ、プラグインやコンポーネントを最新の状態に保てるようになります。また、良くテストされた投稿を作成するために時間を割いてくださったコンテンツ製作者へ資金の提供をします。
 
 <p>
 We have separated some rewards for our Open Collective supporters.
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:63]: Did you miss the translation?

File: src/jp/project/how-to-contribute.md [hunk 6]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+質問に回答したり、経験を共有することで、あなたの手助けを頂ければ幸いです。
 
-We are also daily answering questions in the Vue.js community on Discord ([Vue Land](https://discord.gg/vue)), join us.
+また、 Discord での Vue.js コミュニティで日々質問に回答しています ([Vue Land](https://discord.gg/vue)), 是非、ご参加ください。
 
-## Sharing
+## 共有
 
-When you share, you are helping the web community to understand more about accessibility through content.
+共有することで、ウェブコミュニティがコンテンツを通じてアクセシビリティに関する理解をより深めるのを手助けすることができます。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:50]:
```suggestion
共有することで、Web コミュニティがコンテンツを通じてアクセシビリティに関する理解を深める手助けをすることができます。
```

How about this?

File: src/jp/project/how-to-contribute.md [hunk 7]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
 
-[Vue a11y](/) focuses specifically on helping the Vue.js community by creating content, plugins, and components that make it easier on a daily basis.
+[Vue a11y](/) は特に、コンテンツやプラグイン、そしてコンポーネントを作成することでVue.js コミュニティを支援しており、日ごろから Vue.js を簡単にすることに力を入れています。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:52]:
```suggestion
[Vue a11y](/) は特に、コンテンツやプラグイン、そしてコンポーネントを作成することで Vue.js コミュニティを支援しており、日ごろから Vue.js を簡単にすることに力を入れています。
```

[nits]

File: src/jp/project/how-to-contribute.md [hunk 8]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
 
-We count on your support and the support of the community to make it reach your friends and other developers.
+私たちは、あなたの友人や他の開発者に届けるために、あなたのサポートとコミュニティのサポートを頼りにしています。
 
-## Supporting
+## 支援
 
-The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+このプロジェクトは、Vue.jsやアクセシビリティを愛し、 Web がより包括的になることの重要性を理解している人々によって支えられています。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:58]:
```suggestion
このプロジェクトは、Vue.js やアクセシビリティを愛し、 Web がより包括的になることの重要性を理解している人々によって支えられています。
```

[nits]

File: src/jp/project/how-to-contribute.md [hunk 9]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+まず、あなたがこのページに訪れ、このプロジェクトに対する貢献について考えてくださることに感謝します。
 
-## Development
+## 開発
 
-To contribute to the development of the site, you can initially open an issue on GitHub and report an error or propose a new feature.
+このサイトの開発に貢献するためには、まず最初に Github 上で issue を開き、エラー報告や新機能の提案をすることができます。
 
-If you are willing to develop, there is no mystery, just open a Pull Request following the template.
+もし、あなたが貢献しようと考えるなら、 特別なことは何も必要ありません、テンプレートに沿って Pull Request を開くだけです。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:14]:
```suggestion
もし、あなたが貢献しようと考えるなら、特別なことは何も必要ありません、テンプレートに沿って Pull Request を開くだけです。
```

trim whitespace

File: src/jp/project/how-to-contribute.md [hunk 10]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+## 開発
 
-To contribute to the development of the site, you can initially open an issue on GitHub and report an error or propose a new feature.
+このサイトの開発に貢献するためには、まず最初に Github 上で issue を開き、エラー報告や新機能の提案をすることができます。
 
-If you are willing to develop, there is no mystery, just open a Pull Request following the template.
+もし、あなたが貢献しようと考えるなら、 特別なことは何も必要ありません、テンプレートに沿って Pull Request を開くだけです。 
 
-This website uses [Vuepress](https://vuepress.vuejs.org/), [TailwindCSS](https://tailwindcss.com/), and [Vue composition API](https://composition-api.vuejs.org/).
+このサイトは [Vuepress](https://vuepress.vuejs.org/), [TailwindCSS](https://tailwindcss.com/), と [Vue composition API](https://composition-api.vuejs.org/) を使用しています。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:16]:
```suggestion
このサイトは [Vuepress](https://vuepress.vuejs.org/)、[TailwindCSS](https://tailwindcss.com/)、と [Vue composition API](https://composition-api.vuejs.org/) を使用しています。
```

File: src/jp/project/how-to-contribute.md [hunk 11]
```diff
@@ -3,61 +3,61 @@ description: Find out how to contribute to Vue Accessibility project
...
+私たちは、あなたの友人や他の開発者に届けるために、あなたのサポートとコミュニティのサポートを頼りにしています。
 
-## Supporting
+## 支援
 
-The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+このプロジェクトは、Vue.jsやアクセシビリティを愛し、 Web がより包括的になることの重要性を理解している人々によって支えられています。
 
-Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+皆様からのご支援は非常に重要であり、コンテンツ作成にさらに時間を割くことができ、プラグインやコンポーネントを最新の状態に保てるようになります。また、良くテストされた投稿を作成するために時間を割いてくださったコンテンツ製作者へ資金の提供をします。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:60]:
```suggestion
皆様からのご支援は非常に重要であり、コンテンツ作成にさらに時間を割くことができ、プラグインやコンポーネントを最新の状態に保てるようになります。また、十分に検証された投稿を作成するために時間を割いてくれたコンテンツ製作者に資金を提供しています。
```

How about this?

File: src/jp/project/how-to-contribute.md (hunk 5 above)
Comment (yamanoku) [reply src/jp/project/how-to-contribute.md:63]: Please https://github.com/vue-a11y/vue-a11y.com/pull/39/files#diff-319aeeb595aaf58842f44f0daec5863eea253a43b0d86e90c45ad21bf0f14c23L66 alt text

**Review by yamanoku — COMMENTED** [review]
@kazuki-komori Thx, good contribution! I reviewed it!

File: src/jp/project/how-to-contribute.md (hunk 5 above)
Comment (kazuki-komori) [reply src/jp/project/how-to-contribute.md:63]: @yamanoku 
> We have separated some rewards for our Open Collective supporters.

How can I translate this sentence??
Please, give me some ideas 🙇

**Comment by kazuki-komori** [comment]
@yamanoku 
Thank you for your thoughtful review.
I fixed some wrong translations. 👍

File: src/jp/project/how-to-contribute.md [hunk 12]
```diff
@@ -3,67 +3,67 @@ description: Find out how to contribute to Vue Accessibility project
...
 
-The project is supported by people who love Vue.js and accessibility, and who know the importance of the web being more inclusive.
+このプロジェクトは、Vue.js やアクセシビリティを愛し、Web がより包括的になることの重要性を理解している人々によって支えられています。
 
-Your support is very important so that we can devote more time to creating content, keeping plugins and components up to date, in addition to funding content creators for their time dedicated to creating robust and well-tested posts.
+皆様からのご支援は非常に重要であり、コンテンツ作成にさらに時間を割くことができ、プラグインやコンポーネントを最新の状態に保てるようになります。また、十分に検証された投稿を作成するために時間を割いてくれたコンテンツ製作者に資金を提供しています。
 
 <p>
-We have separated some rewards for our Open Collective supporters.
+Open Collective のサポーターの方への報酬をいくつかに分けています。
```
Comment (yamanoku) [inline src/jp/project/how-to-contribute.md:63]:
```suggestion
Open Collective サポーターの方々のためにいくつかの支援を分けています。
```

How about this ?
`報酬` => `支援`

File: src/jp/project/how-to-contribute.md (hunk 5 above)
Comment (yamanoku) [reply src/jp/project/how-to-contribute.md]: please check it !
https://github.com/vue-a11y/vue-a11y.com/pull/39/files#r563020598

**Comment by yamanoku** [comment]
@kazuki-komori
I checked fixed document, thanks! Please check one more place.

**Comment by kazuki-komori** [comment]
@yamanoku 
I fixed the translation 👌 
I adopted the meaning `支援`
please check it 🙇

**Review by yamanoku — APPROVED** [review]
LGTM !!!
Thanks, good translation!

**Comment by ktquez** [comment]
Is it ok to merge?

**Comment by ktquez** [comment]
Done!

Thanks @kazuki-komori and @yamanoku

**Pull request merged**